src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink';
+
+export const PeopleTable = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<boolean>(false);
+
+  const { slug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setError(false);
```

Comment:
You have 'false' by default, don't have to set it again

Reply:
+1